en/telemetry/esp32_wifi_module.md

```diff
@@ -1,169 +1,159 @@
+
+::: info
+ESP-NOW requires ESP32 devices on the GCS side as well as on the autopilot side.
+:::
+
+## Features
+
+- Bidirectional: serial-to-WiFi, serial-to-WiFi Long-Range (LR), serial-to-ESP-NOW link
+- Support for MAVLink, MSP, LTM or any other payload using transparent option
+- Affordable, Reliable & low latency
+- Weight: <8 g
+- Up to 150m range using standard WiFi
+- Up to 1km of range using ESP-NOW or Wi-Fi LR Mode - sender & receiver must be ESP32 with LR-Mode enabled
+- Fully encrypted in all modes including ESP-NOW broadcasts secured using AES-GCM 256 bit!
+- Support for drone swarms of almost any size using ESP-NOW with its custom & encrypted broadcast mode
+- Supported by: QGroundControl, Mission Planner, mwptools, impload etc.
+- Easy to set up: Power connection + UART connection to flight controller
+- Fully configurable through an easy-to-use web interface
+- Parsing of LTM & MSPv2 for more reliable connection and less packet loss
+- Parsing of MAVLink with the injection of Radio Status packets for the display of RSSI in the GCS
+- Fully transparent telemetry down-link option
+
+::: info
+[You can find all the latest information on the current version of _DroneBridge for ESP32_ on the official website.](https://dronebridge.github.io/ESP32/)
+:::
 
 ## Recommended Hardware
 
-_DroneBridge for ESP32_ can run on almost every ESP32 development board.
-Boards and modules with an external antenna connector are recommended, since these will offer more range.
+_DroneBridge for ESP32_ can run on almost all ESP32 development boards.
+Boards and modules with an external antenna connector are recommended since these will offer more range.
 
 :::warning
-Many ESP32 modules support 3.3V and 5V power supply input, while some flight controllers (e.g. Pixhawk 4) output at 5V.
+Some ESP32 modules support 3.3V power supply input, while many flight controllers output at 5V (e.g. Pixhawk 4).
 You will need to check compatibility and step down the voltage if needed.
 :::
 
-Modules and DevKits that accept 3.3V or 5V power supply:
+::: tip
+The _DroneBridge for ESP32_ project recommends using [Officially Supported and Tested Boards](https://dronebridge.github.io/ESP32/).
+These are small, low in price, have Pixhawk-standard port for connecting to Pixhawk flight controllers, and come with _DroneBridge for ESP32_ firmware pre-installed.
+:::
+
+Other ESP32 Boards that have been tested with earlier versions of _DroneBridge for ESP32_:
 
 - [AZ-Delivery — ESP-32 DevKit C](https://www.az-delivery.de/en/products/esp-32-dev-kit-c-v4)
 - [TinyPICO — ESP32 Development Board - V2](https://www.adafruit.com/product/4335)
 - [Adafruit HUZZAH32 — ESP32 Feather Board](https://www.adafruit.com/product/3405)
 - [Adafruit AirLift — ESP32 WiFi Co-Processor Breakout Board](https://www.adafruit.com/product/4201) (requires FTDI adapter for flashing firmware)
 - [Adafruit — HUZZAH32](https://www.adafruit.com/product/4172) (requires FTDI adapter for flashing firmware)
```

Comment:
I have modified this to add back the list. I'm more than happy to recommend the official boards, but not everyone can ship from EU and we want the guide to be useful even if there are shortages of those.